File: C++/Estrutura Sequencial/Medidas.cpp
#include <iostream>
#include <iomanip>
#include <string>
#include <climits>
#include <cmath>

using namespace std;

int main()
{

    double a, b, c, areaQ, areaTri, areaTra;

    cout << "Digite a medida A: ";
    cin >> a;
    cout << "Digite a medida B: ";
    cin >> b;
    cout << "Digite a medida C: ";
    cin >> c;

    areaQ = pow(a, 2);
    areaTri = (a * b) / 2;
    areaTra = ((a + b) * c) / 2;

    cout << fixed << setprecision(4);
    cout << "AREA DO QUADRADO = " << areaQ << endl;
    cout << "AREA DO TRIANGULO = " << areaTri << endl;
    cout << "AREA DO TRAPEZIO = " << areaTra << endl;

    return 0;
}

File: Python/Estrutura Sequencial/Medidas.py
import math

a: float; b: float; c: float; areaQuad: float; areaTri: float; areaTrap: float

a = float(input("Digite a media A: "))
b = float(input("Digite a media B: "))
c = float(input("Digite a media C: "))


areaQuad = math.pow(a, 2.0)
areaTri = (a * b) / 2
areaTrap = ((a + b) * c) / 2

print(f"AREA DO QUADRADO = {areaQuad:.4F}")
print(f"AREA DO TRIANGULO = {areaTri:.4F}")
print(f"AREA DO TRAPEZIO = {areaTrap:.4F}")

